rust/foxglove/src/websocket/ws_protocol/parameter.rs

```diff
@@ -80,7 +80,11 @@ impl Parameter {
 
     /// Creates a new parameter with a string value.
     pub fn string(name: impl Into<String>, value: impl Into<String>) -> Self {
-        Self::byte_array(name, value.into().into_bytes())
+        Self {
+            name: name.into(),
+            r#type: None,
+            value: Some(ParameterValue::String(value.into().into_bytes())),
```

Comment:
Is it correct that we're converting to bytes and then base64-encoding? Why would I use a string parameter and not a byte array in that case? Maybe this should be using the utf8 string directly?

Reply:
oh, I see #382 now